static/openapi/est-spec.json

```diff
@@ -0,0 +1,1649 @@
+            "description": "Photoniq EST service variable deleted successfully"
+          },
+          "500": {
+            "description": "Error deleting Photoniq EST variable"
+          }
+        }
+      }
+    }
+  },
+  "components": {
+    "schemas": {
+      "MetricsResponse": {
+        "type": "object",
+        "properties": {
+          "activityMetrics": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/ActivityMetrics"
+            }
+          },
+          "type": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/TotalMetrics"
+            }
+          }
+        }
+      },
+      "ActivityMetrics": {
+        "type": "object",
+        "properties": {
+          "customerId": {
+            "type": "string",
+            "description": "ID of the customer."
+          },
+          "componentName": {
+            "type": "string",
+            "description": "Name of the component, example google-analytics."
+          },
+          "eventType": {
+            "type": "string",
+            "description": "Type of event, example pageView or client."
+          },
+          "eventName": {
+            "type": "string",
+            "description": "Name of event, example click | scroll | mousedown etc."
+          },
+          "count": {
+            "type": "number",
+            "description": "Number of events."
+          }
+        }
+      },
+      "TotalMetrics": {
+        "type": "object",
+        "properties": {
+          "totalRequests": {
+            "type": "number",
+            "description": "Number of total requests between from and to time."
+          },
+          "totalBytes": {
+            "type": "number",
+            "description": "Number of total bytes used between from and to time."
+          }
+        }
+      },
+      "ErrorMetrics": {
+        "type": "object",
+        "properties": {
+          "customerId": {
+            "type": "string",
+            "description": "ID of the customer."
+          },
+          "errorName": {
+            "type": "string",
+            "description": "Name of the error occured."
+          },
+          "count": {
+            "type": "number",
+            "description": "Number of errors."
+          }
+        }
+      },
+      "UsageMetrics": {
+        "type": "object",
+        "properties": {
+          "customerId": {
+            "type": "string",
+            "description": "ID of the customer."
+          },
+          "usedRequests": {
+            "type": "number",
+            "description": "Number of requests made by the customer."
+          },
+          "usedBytes": {
+            "type": "number",
+            "description": "Number of bytes used."
+          }
+        }
+      }
+    }
+  },
+  "tags": [
+    {
+      "name": "Activity Metrics",
+      "description": "All the Photoniq EST activity metrics endpoints."
```

Comment:
```suggestion
      "description": "All activity metrics endpoints."
```

Reply:
As @joubert-macrometa pointed out, other APIs do not constantly repeat the product name. We shouldn't here. Leave it in the change history and customer ID, remove it everywhere else.